samples/excel/90-scenarios/currency-converter.yaml

```diff
@@ -0,0 +1,178 @@
+order: 5
+id: excel-scenarios-currency-converter
+name: Currency Converter
+description: 'Uses an exchange rate API to convert currency values based on their original transaction times.'
+author: cakriwut
+host: EXCEL
+api_set:
+    ExcelApi: '1.4'
+script:
+    content: |
+        /*
+        * Copyright (c) Riwut Libinuko. All rights reserved. Licensed under the MIT license.
+        */
+
+        declare let moment: any;
+        const tableName = "TransactionTable";
+
+        let tblSetup: Record<string, number> = {};
```

Comment:
Nit: Can we rename `tblSetup` to `tableSetup`?

Reply:
Updated.